oap-stdlib/src/test/java/oap/application/ApplicationConfigurationTest.java

```diff
@@ -70,14 +70,13 @@ public void testProfiles() {
         assertThat( ac.getProfiles() ).isEmpty();
         ac.reset();
 
-        ac.profiles.add( "-with-file" );
         ac.profiles.add( "test" );
 
         System.setProperty( "OAP_PROFILE_with__file", "1" );
         Env.set( "OAP_PROFILE_with_2Dfile", "on" );
         Env.set( "OAP_PROFILE_test", "false" );
 
-        assertThat( ac.getProfiles() ).containsOnly( "with_file", "with-file", "-test" );
```

Comment:
1. Could you explain me why exactly these sequence was in the test (how ENV variable controlles)?
2. Why test profile started to be enabled after changes?
3. Why sequence in `containsOnly` changed?

Reply:
ENV variables allow to swith on/off profiles. 
Sequence doesn't matter in contains comparison. 
I've put test in the first place because it 1st appeared in the test and stays the 1st. The rest profiles stayed the same. "-test" is skipped now.